src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,12 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream().map(str -> str.split(","))
```

Comment:
Isn't it better to extract ```","``` to sth like ```NUMBERS_SEPARATOR``` constant?

Reply:
Thanks for your advice! Done.